wrapanapi/rest_client.py

```diff
@@ -31,6 +32,8 @@ def __init__(self, hostname, auth, protocol="https", port=6443, entry='api/v1',
             entry: Entry point of the REST API
             verify: 'True' if we want to verify SSL, 'False' otherwise
         """
+        logging.basicConfig(level=logging.INFO)
```

Comment:
it is not a good to call `logging.basicConfig` in a library, it just confuses any app using it,
in addition calling it in a `__init__` method is really pushing it

the correct way is to fetch the root logger for the package adding a a `NullHandler` so logging wont complain
in addition to linking up the logging handling of an application in order to get all the desired logging

i believe we need to connect the logging handlers in the integration tests repo to this logging root as well in order to get all the logging data we need/want after we bump to a version containing this addition